StratisCore.UI/src/app/wallet/smart-contracts/components/modals/transaction/transaction.component.html

```diff
@@ -17,30 +17,30 @@ <h5 class="modal-title">{{title}}</h5>
                 </div>
                 <div class="form-group clearfix" data-toggle="tooltip" data-placement="right" title="The amount of {{ coinUnit }} to send with this transaction">
                     <label for="amount">Amount</label>
-                    <input formControlName="amount" id="amount" type="number" step="0.01" class="form-control" placeholder="Amount" [class.is-invalid]="amount.invalid && (amount.dirty || amount.touched)">
+                    <input formControlName="amount" id="amount" type="number" step="0.01" class="form-control" placeholder="Amount" [class.is-invalid]="amount.invalid && (amount.dirty || amount.touched)" min="0">
                     <div *ngIf="amount.errors" class="invalid-feedback">
-                        <p *ngIf="amount.errors.amountError">Amount must be less than your balance</p>
+                        <p *ngIf="amount.errors.amountError">The amount you have entered exceeds balance available at the sender address</p>
                     </div>
                 </div>
                 <div class="form-group clearfix" data-toggle="tooltip" data-placement="right" title="The fee to send with this transaction ({{ coinUnit }}/bytes)">
                     <label for="feeAmount">Fee</label>
-                    <input formControlName="feeAmount" id="feeAmount" type="number" step="0.01" class="form-control" placeholder="Fee" [class.is-invalid]="feeAmount.invalid && (feeAmount.dirty || feeAmount.touched)">
+                    <input formControlName="feeAmount" id="feeAmount" type="number" step="0.01" class="form-control" placeholder="Fee" [class.is-invalid]="feeAmount.invalid && (feeAmount.dirty || feeAmount.touched)" min="0">
                     <div *ngIf="feeAmount.errors" class="invalid-feedback">
                         <p *ngIf="feeAmount.errors.feeError">Fee must be less than your balance</p>
                     </div>
                 </div>
                 <div class="row">
                     <div class="form-group col-6 clearfix" data-toggle="tooltip" data-placement="right" title="The gas price to use with this transaction ({{ coinUnit }}/gas)">
                         <label for="gasPrice">Gas price</label>
-                        <input formControlName="gasPrice" id="gasPrice" type="number" class="form-control" placeholder="Gas Price" [class.is-invalid]="gasPrice.invalid && (gasPrice.dirty || gasPrice.touched)">
+                        <input formControlName="gasPrice" id="gasPrice" type="number" class="form-control" placeholder="Gas Price" [class.is-invalid]="gasPrice.invalid && (gasPrice.dirty || gasPrice.touched)" min="0">
```

Comment:
min=0 should actually be checked in transaction.component.ts using a validator like we do throughout the rest of the wallet.
This is beyond the scope of the current issue though, but needs changing throughout the entire smart contracts component.